<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="1" hostname="suraj" failures="2" tests="24" name="Test" time="47.492" errors="0" timestamp="2022-10-06T17:05:34 IST">
  <testcase classname="com.vyasa.automation.testing.TestNGAnnotations" name="testMethod" time="0.004"/>
  <testcase classname="com.vyasa.automation.testing.TestNGAnnotations1" name="testMethod" time="0.001"/>
  <testcase classname="com.vyasa.automation.testing.DataProviderDemo" name="verifUserLogon" time="2.718"/>
  <testcase classname="com.vyasa.automation.testing.DataProviderDemo" name="verifUserLogon" time="2.767"/>
  <testcase classname="com.vyasa.automation.testing.DataProviderDemo" name="verifUserLogon" time="2.422"/>
  <testcase classname="com.vyasa.automation.testing.DataProviderDemo" name="verifUserLogon" time="2.682"/>
  <testcase classname="com.vyasa.automation.testing.OpenBrowers" name="verifyCartDisplayed" time="1.11"/>
  <testcase classname="com.vyasa.automation.testing.OpenBrowers" name="verifyHomePageLoaded" time="0.065"/>
  <testcase classname="com.vyasa.automation.testing.DemoTestNG" name="TestCase1" time="0.001">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.vyasa.automation.testing.DemoTestNG.TestCase1(DemoTestNG.java:14)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- TestCase1 -->
  <testcase classname="com.vyasa.automation.testing.HomePageAutomation" name="verifyHomePageLoaded" time="0.036"/>
  <testcase classname="com.vyasa.automation.testing.DemoTestNG" name="TestCase5" time="0.0"/>
  <testcase classname="com.vyasa.automation.testing.HomePageAutomation" name="verifyNewArrivalsDisplayed" time="0.145"/>
  <testcase classname="com.vyasa.automation.testing.DemoTestNG" name="TestCase12" time="0.0"/>
  <testcase classname="com.vyasa.automation.testing.HomePageAutomation" name="verifyHomePageWithThreeSliders" time="0.043"/>
  <testcase classname="com.vyasa.automation.testing.HomePageAutomation" name="verifyHomePageWithThreeArrivals" time="0.029"/>
  <testcase classname="com.vyasa.automation.testing.DemoTestNG" name="TestCase11" time="0.001"/>
  <testcase classname="com.vyasa.automation.testing.HomePageAutomation" name="verifyHomePageWithThreeArrivalsNavigation" time="6.009">
    <failure type="org.openqa.selenium.StaleElementReferenceException" message="stale element reference: element is not attached to the page document
  (Session info: chrome=105.0.5195.127)
For documentation on this error, please visit: https://www.seleniumhq.org/exceptions/stale_element_reference.html
Build info: version: &amp;apos;3.141.59&amp;apos;, revision: &amp;apos;e82be7d358&amp;apos;, time: &amp;apos;2018-11-14T08:17:03&amp;apos;
System info: host: &amp;apos;SURAJ&amp;apos;, ip: &amp;apos;192.168.239.134&amp;apos;, os.name: &amp;apos;Windows 10&amp;apos;, os.arch: &amp;apos;amd64&amp;apos;, os.version: &amp;apos;10.0&amp;apos;, java.version: &amp;apos;18.0.1.1&amp;apos;
Driver info: org.openqa.selenium.chrome.ChromeDriver
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 105.0.5195.127, chrome: {chromedriverVersion: 105.0.5195.52 (412c95e51883..., userDataDir: C:\Users\admin\AppData\Loca...}, goog:chromeOptions: {debuggerAddress: localhost:57392}, javascriptEnabled: true, networkConnectionEnabled: false, pageLoadStrategy: normal, platform: WINDOWS, platformName: WINDOWS, proxy: Proxy(), setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:virtualAuthenticators: true}
Session ID: 61e00b60ed46b56e92527a9c46dab320">
      <![CDATA[org.openqa.selenium.StaleElementReferenceException: stale element reference: element is not attached to the page document
  (Session info: chrome=105.0.5195.127)
For documentation on this error, please visit: https://www.seleniumhq.org/exceptions/stale_element_reference.html
Build info: version: '3.141.59', revision: 'e82be7d358', time: '2018-11-14T08:17:03'
System info: host: 'SURAJ', ip: '192.168.239.134', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '18.0.1.1'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 105.0.5195.127, chrome: {chromedriverVersion: 105.0.5195.52 (412c95e51883..., userDataDir: C:\Users\admin\AppData\Loca...}, goog:chromeOptions: {debuggerAddress: localhost:57392}, javascriptEnabled: true, networkConnectionEnabled: false, pageLoadStrategy: normal, platform: WINDOWS, platformName: WINDOWS, proxy: Proxy(), setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:virtualAuthenticators: true}
Session ID: 61e00b60ed46b56e92527a9c46dab320
at org.openqa.selenium.remote.http.W3CHttpResponseCodec.createException(W3CHttpResponseCodec.java:187)
at org.openqa.selenium.remote.http.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:122)
at org.openqa.selenium.remote.http.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:49)
at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:158)
at org.openqa.selenium.remote.service.DriverCommandExecutor.execute(DriverCommandExecutor.java:83)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:552)
at org.openqa.selenium.remote.RemoteWebElement.execute(RemoteWebElement.java:285)
at org.openqa.selenium.remote.RemoteWebElement.getAttribute(RemoteWebElement.java:134)
at com.vyasa.automation.testing.HomePageAutomation.verifyHomePageWithThreeArrivalsNavigation(HomePageAutomation.java:53)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- verifyHomePageWithThreeArrivalsNavigation -->
  <testcase classname="com.vyasa.automation.testing.DemoTestNG" name="TestCase3" time="0.0"/>
  <testcase classname="com.vyasa.automation.testing.DemoTestNG" name="TestCase4" time="0.001"/>
  <testcase classname="com.vyasa.automation.testing.DemoTestNG" name="TestCase6" time="0.0"/>
  <testcase classname="com.vyasa.automation.testing.DemoTestNG" name="TestCase7" time="0.0"/>
  <testcase classname="com.vyasa.automation.testing.DemoTestNG" name="TestCase8" time="0.001"/>
  <testcase classname="com.vyasa.automation.testing.DemoTestNG" name="TestCase9" time="0.006"/>
  <testcase classname="com.vyasa.automation.testing.DemoTestNG" name="TestCase10" time="0.001"/>
  <testcase classname="com.vyasa.automation.testing.DemoTestNG" name="TestCase2" time="0.0">
    <ignored/>
  </testcase> <!-- TestCase2 -->
</testsuite> <!-- Test -->
